src/router/index.js

```diff
@@ -22,23 +22,39 @@ export default function({store}) {
   })
 
   Router.beforeEach((to, _, next) => {
-    // routing page by authenticate token
-    if (isAuth()) {
+    let userType = getUserType()
+    // user is CUSTOMER
+    if (userType === 'c') {
       if (to.path === '/customer') {
         next()
         // prevent route to register page
-      } else if (to.path === '/customer/register' || to.path === '/customer/login') {
+      } else if (to.path === '/customer/register' || to.path === '/customer/login' || to.path.indexOf('admin') > 0) {
         next('/customer')
       } else next()
-    } else {
-      // handle routing part
-      if (
-        to.path === '/customer/register' ||
-        to.path === '/customer/login' ||
-        (to.path !== '/customer/login' && to.path !== '/customer') // prevent route to customer page
-      ) {
-        next()
-      } else next('/customer/login')
+
+      // user is ADMIN
+    } else if (userType === 'a') {
+      // route to admin home page
+      if (to.path === '/admin') next()
+      else next('/admin')
+
+      // user is ANONYMOUS
+    } else if (userType === false) {
+      // recognize page of user by path to route
+      if (to.path.indexOf('customer') > -1) {
+        // route to CUSTOMER login page
+        if (
+          to.path === '/customer/register' ||
+          to.path === '/customer/login' ||
+          (to.path !== '/customer/login' && to.path !== '/customer') // prevent route to customer page
+        ) {
+          next()
+        } else next('/customer/login')
+      } else {
+        // routing to ADMIN login page
+        if (to.path === '/admin/login') next()
```

Comment:
if (to.path === '/admin/login') next()
--> same with : **else next('/admin/login')**
??